pennylane/tape/qnode.py

```diff
@@ -445,6 +444,30 @@ def __call__(self, *args, **kwargs):
             return __import__(res_type_namespace).numpy.squeeze(res)
         return __import__(res_type_namespace).squeeze(res)
 
+    def metric_tensor(self, *args, diag_approx=False, only_construct=False, **kwargs):
+        """Evaluate the value of the metric tensor.
+
+        Args:
+            args (tuple[Any]): positional arguments
+            kwargs (dict[str, Any]): auxiliary arguments
+            diag_approx (bool): iff True, use the diagonal approximation

```

Comment:
```suggestion
            diag_approx (bool): Iff True, use the diagonal approximation

```

Reply:
Wait, the convention is: capital if sentence with period, lower case if not, right?

Reply:
yep!